DTAdmin/MainTabBarController.swift

```diff
@@ -80,10 +80,10 @@ class MainTabBarController: UITabBarController, UITabBarControllerDelegate {
 //MARK: UITabBarControllerDelegate method
     func tabBarController(_ tabBarController: UITabBarController, didSelect viewController: UIViewController) {
         if let title = viewController.title {
-            print("Selected \(title)")
+            //print("Selected \(title)")
```

Comment:
Please remove commented dead code or provide good justification why have you commented it.